Akinmyde/library-backend
# Chore - sequelize setup

__What does this PR do?__
-  This PR sets up sequelize connections
__Description of Task to be completed?__
-  install `npm i sequelize-cli sequelize pg pg-hstore`
- setup sequelize configuration
- setup environment variables
- setup config file to manage node environments
- create postgres database
- connect sequelize to the database
__How should this be manually tested?__
- clone the repository
- git checkout to `ch-sequelize-setup` branch
- install dependencies `npm i`
- run `npm run start:conn` script to confirm connections to the appropriate database
__What are the relevant pivotal tracker stories?__

__Screenshots__
